### ISSUE TITLE
[BUG] PartBeamFormer crashes.

### ISSUE THREAD

**Issue by KasumiNova** [AE2-UEL/NAE2#46]
Can't quite reproduce it, can only assume it's an occasional problem, but it's causing my saves to not work properly.
Using the latest NAE2.

![图片](https://github.com/AE2-UEL/NAE2/assets/48862050/07ff3b6e-ddc0-4722-b757-84c759f24d5f)

**Comment by NotMyWing** [comment]
The log suggests that `listenerLinkedList` is null, which should never be the case since the code is unreachable if there's no connection to begin with. Please post the entire log if possible.

https://github.com/AE2-UEL/NAE2/blob/8f315f6aaa2938291da9222d3ad8d3ec378b5570/src/main/java/co/neeve/nae2/common/parts/implementations/PartBeamFormer.java#L251

**Comment by KasumiNova** [comment]
https://mclo.gs/KU0CmJC
Not sure if any of this will be useful in solving the problem, but this is the only log.

**Comment by NotMyWing** [comment]
Right, no anomalies in the log. I'll implement an extra safeguard, but in all honesty this should never have happened in the first place.

**Comment by Bili1Gd** [comment]
pause game when you change dimensions and enter game can prevent it maybe

**Comment by MRSlouzk** [comment]
I have the same crash too，it's my crash report here：[https://mclo.gs/7cGDICT](crash-report)，I dont know how it happened，I did not do anything ever when it crashed, it's so sudden.

**Comment by romanfedyniak** [comment]
https://mclo.gs/26pCzmC same problem
